server/api/reviews.go

```diff
@@ -253,6 +263,66 @@ func ReviewsHandler(w http.ResponseWriter, r *http.Request) {
 
 		utils.RespondWithSuccess(w, http.StatusCreated, review)
 
+	case http.MethodDelete:
```

Comment:
Delete is not in CORS allowlist.

Reply:
It's there https://github.com/veritem/bookworms/blob/f377a20c9fa0695e544ab43cf05893b4bd62bcac/server/api/reviews.go#L192